sonata/main.py

```diff
@@ -1234,8 +1234,12 @@ def on_add_item(self, _widget, play_after=False):
             if self.current_tab == self.TAB_LIBRARY:
                 items = self.library.get_path_child_filenames(True)
                 self.mpd.command_list_ok_begin()
-                for item in items:
-                    self.mpd.add(item)
+                for item in items:     ######
+                     if item.endswith('.pls') or item.endswith('.PLS') or \
```

Comment:
This should be extracted into a dedicated function, since I guess we'll have to support something else than ``.pls`` and ``.m3u`` playlists one day.
Also, this could be simplified if you convert your "item" value into lower/upper case.